docs/getting-started-guides/kubeadm.md

```diff
@@ -54,155 +65,209 @@ If you are not constrained, there are some other tools built to give you complet
+  apt-get install -y kubelet kubeadm kubectl kubernetes-cni
+  ```
+
+  If the machine is running CentOS, run:
+
+  ``` bash
+  cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+  [kubernetes]
+  name=Kubernetes
+  baseurl=http://yum.kubernetes.io/repos/kubernetes-el7-x86_64
+  enabled=1
+  gpgcheck=1
+  repo_gpgcheck=1
+  gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+          https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+  EOF
+  setenforce 0
+  yum install -y docker kubelet kubeadm kubectl kubernetes-cni
+  systemctl enable docker && systemctl start docker
+  systemctl enable kubelet && systemctl start kubelet
+  ```
+
+The kubelet is now restarting every few seconds, as it waits in a crashloop for
+kubeadm to tell it what to do.
+
+Note: Disabling SELinux by running `setenforce 0` is required in order to allow
+containers to access the host filesystem, which is required by pod networks for
+example. You have to do this until kubelet can handle SELinux better.
 
 ### (2/4) Initializing your master
 
-The master is the machine where the "control plane" components run, including `etcd` (the cluster database) and the API server (which the `kubectl` CLI communicates with).
-All of these components run in pods started by `kubelet` and the following images are required and will be automatically pulled by `kubelet` if they are absent while `kubeadm init` is initializing your master:
-
-    gcr.io/google_containers/kube-proxy-amd64                v1.5.3         
-    gcr.io/google_containers/kube-controller-manager-amd64   v1.5.3         
-    gcr.io/google_containers/kube-scheduler-amd64            v1.5.3         
-    gcr.io/google_containers/kube-apiserver-amd64            v1.5.3         
-    gcr.io/google_containers/etcd-amd64                      3.0.14-kubeadm 
-    gcr.io/google_containers/kube-discovery-amd64            1.0            
-    gcr.io/google_containers/pause-amd64                     3.0 
-
-Right now you can't run `kubeadm init` twice without tearing down the cluster in between, see [Tear down](#tear-down).
+The master is the machine where the "control plane" components run, including
+etcd (the cluster database) and the API server (which the kubectl CLI
+communicates with). All of these components run in pods started by kubelet and
+the following images are required and will be automatically pulled by kubelet
+if they are absent while `kubeadm init` is initializing your master:
+
+| Image Name | Version |
+|---|---|
+| gcr.io/google_containers/kube-proxy-amd64 | v1.5.3
+| gcr.io/google_containers/kube-controller-manager-amd64 | v1.5.3
+| gcr.io/google_containers/kube-scheduler-amd64 | v1.5.3
+| gcr.io/google_containers/kube-apiserver-amd64 | v1.5.3
+| gcr.io/google_containers/etcd-amd64 | 3.0.14-kubeadm
+| gcr.io/google_containers/kube-discovery-amd64 | 1.0
+| gcr.io/google_containers/pause-amd64 | 3.0
+
+Right now you can't run `kubeadm init` twice without tearing down the cluster in
```

Comment:
Remove "right now" unless we can tell the user something about when this might change.